composer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "wordpress/twentynineteen",
+  "description": "A new Gutenberg-ready theme.",
```

Comment:
This description differs from the description in `package.json` which is currently `Default WP Theme`, taking the description from the themes `style.css` seems best, `Twenty Nineteen`

https://github.com/WordPress/twentynineteen/blob/master/style.css#L3

Reply:
So should `style.css` be updated to match `package.json` description (`Default WP Theme`) or vice versa - should I update `package.json` to say `A new Gutenberg-ready theme.`?

Twentyseventeen had a totally different description

> Twenty Seventeen brings your site to life with immersive featured images and subtle animations. With a focus on business sites, it features multiple sections on the front page as well as widgets, navigation and social menus, a logo, and more. Personalize its asymmetrical grid with a custom color scheme and showcase your multimedia content with post formats. Our default theme for 2017 works great in many languages, for any abilities, and on any device.

I think that this should be changed in both `package.json`, `composer.json` and `style.(s)css` once the theme is ready to be published 🙂

Reply:
Not sure, both the `composer.json` and `package.json` files probably don't require a long description.

Thoughts @allancole @kjellr ?

Reply:
@dingo-d Let's just use the description that we've been putting in the core commit notes:  

> Let Gutenberg shine with this simple, fast, and powerful theme.

This will be revised and expanded for the actual launch, but that'll work for now.